src/main/java/nl/tudelft/ewi/devhub/server/database/entities/Group.java

```diff
@@ -39,7 +39,7 @@
 
 	@NotNull
 	@Column(name = "group_number")
-	private long groupNumber;
+	private Long groupNumber;
```

Comment:
But if it's null you cannot persist it anyway. If NULL is not a valid value, you should use primitives if possible.